static/js/main.js

```diff
@@ -0,0 +1,37 @@
+// var signup_form = document.getElementById('signup_form')
```

Comment:
Try not to push commented out code. If you don't need it then there's no point in pushing it to github

Reply:
iam going to check it out